### PR TITLE
Use secret store always

### PR DESCRIPTION
**Pull request by patrickxb**
This PR removes the option for the user to turn on/off secret store.

### CODE REVIEW

**Comment by patrickxb** [comment]
This needs UI work in CORE-3542 before it can be merged.

And fixes in CORE-3539 will be nice to have as well.

**Comment by maxtaco** [comment]
I think the CI failure is an actual failure...

File: go/libkb/passphrase_helper.go [hunk 1]
```diff
@@ -8,14 +8,19 @@ import (
 	keybase1 "github.com/keybase/client/go/protocol"
 )
 
-func GetKeybasePassphrase(g *GlobalContext, ui SecretUI, username, retryMsg string, allowSecretStore bool) (keybase1.GetPassphraseRes, error) {
-	arg := DefaultPassphraseArg(g, allowSecretStore)
+func GetKeybasePassphrase(g *GlobalContext, ui SecretUI, username, retryMsg string) (keybase1.GetPassphraseRes, error) {
+	arg := DefaultPassphraseArg(g, true)
```
Comment (maxtaco) [inline go/libkb/passphrase_helper.go]: can you just remove the arg?

**Comment by maxtaco** [comment]
Ok, great! 👍

**Comment by patrickxb** [comment]
Note to self: Don't merge this until CORE-3542 complete

**Pull request merged**